src/pages/Game.tsx

```diff
@@ -200,15 +202,13 @@ const Game: React.FC<props> = ({
             isRevealing={isRevealing}
             currentRowClassName={currentRowClass}
           />
-          <p className="text-center text-4xl text-black dark:text-white">
-            {verseText}
-          </p>
-          <a
-            href={referenceUrl}
-            className="text-center text-black dark:text-white"
+          <button
+            onClick={() => setIsHintModalOpen(true)}
+            type="button"
+            className={`mx-auto mt-2 inline-flex w-20 items-center justify-center rounded-md border-2 border-slate-200 px-3 py-1.5 text-center align-middle text-sm font-medium shadow-sm hover:bg-gray-300 focus:outline-none focus:ring-2 focus:ring-slate-500 focus:ring-offset-2 disabled:border-gray-200 disabled:bg-gray-500  dark:border-slate-600 dark:bg-slate-900 dark:text-white dark:hover:bg-slate-700 sm:text-base`}
```

Comment:
this is a lot of classes to inline
- perhaps save them to a variable?
- do we really need this many?

Reply:
I don't know Tailwind conventions but it seems less readable to me to move the classes further away from where they're used. Is that the typical way of handling this in Tailwind?

Reply:
It's normal to see a long list of classes defined in a variable so the markup is cleaner.
The reason I pointed this out, is seeing this many classes for a static element resembles a code smell to me. I mostly wanted to ask the question "are all of these classes necessary?"

Reply:
Good question. 

@djyarnell0 Where did these come from? Were they copied from another component? 

Can you test these and determine if any of them are unnecessary?

Reply:
Also, @kadenbking do you see some that seem especially questionable?